layouts/FormEditor/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { useState } from 'react';
+import Nav from '@/components/Nav';
+import { Box, Flex } from '@chakra-ui/react';
+import Toolbar, { ToolType } from '@/layouts/FormEditor/components/Toolbar';
+
+const FormEditorLayout: React.FC = ({ children }) => {
+  const [selectedTool, setSelectedTool] = useState(ToolType.BUILDER);
```

Comment:
This is controlled here because we're going to have to control the active "panel" in the layout